Composer/packages/client/src/components/CreationFlow/v2/CreationFlow.tsx

```diff
@@ -181,7 +181,10 @@ const CreationFlowV2: React.FC<CreationFlowProps> = () => {
           path="create/:runtimeLanguage/:templateId"
           updateFolder={updateFolder}
           onCurrentPathUpdate={updateCurrentPath}
-          onDismiss={handleDismiss}
+          onDismiss={() => {
+            TelemetryClient.track('CreationCancelled');
```

Comment:
why not just put this in handleDismiss instead of calling it 3x

Reply:
Because HandleDismiss is called to dismiss 'save as' modal and 'open' modal so putting it in their would fire the 'creationCancelled' telemetry when creation was not cancelled.